open-company/open-company-web
# WST updates

Card: https://trello.com/c/WbdtEwlt

To test:
- use a team with at least 4 or 5 users
- create a new post and publish it
- expand the post and click on the viewers stats
- do you see all users with the same color?
- do you see your name in bold at the top?
- expand the filter dropdown: choices are not dimmed out?
- do you see the Send all unopened button at the top?
- click it
- did it prompt you to send a reminder to x users?
- accept
- no errors?
- check the network request succeeded
- now get a post that was opened by all but one user in the team
- expand the post and click on Viewers
- do you NOT see the send to all button?
- check mobile for regressions
- button on mobile only says Send all since space is limited (the prompt after you tap will explain better)
